app/helpers/deadline_helper.rb

```diff
@@ -8,6 +8,9 @@ module DeadlineHelper
   DEADLINE_TYPE_SIGN_UP = 7
   DEADLINE_TYPE_TEAM_FORMATION = 8
 
+  TOPIC_DEADLINE_TYPE = 'TopicDueDate'
+  ASSNT_DEADLINE_TYPE = 'AssignmentDueDate'
```

Comment:
Please change ASSNT_DEADLINE_TYPE  to ASSIGNMENT_DEADLINE_TYPE. It is easier to read and understand since this is a constant variable and name should fully describe what is being stored in that variable.